sdks/go/pkg/beam/core/runtime/exec/pardo.go

```diff
@@ -79,6 +86,7 @@ func (n *ParDo) Up(ctx context.Context) error {
 	return nil
 }
 
+// StartBundle starts processing for this bundle and does before bundle processing operations.
```

Comment:
Sentence sounds awkward around "and does before bundle processing operations." Missing word?

Reply:
Nvm, just realized how this is meant to be read. Maybe hyphenate it so it says "pre-bundle-processing" to make it less ambiguous? It looks a bit funny like that, but it's preferable to being difficult to parse.

Reply:
Cleaned it up. Arguably a bundle hasn't "started" until after this method returns.